src/signal.rs

```diff
@@ -477,4 +478,59 @@ mod tests {
         // forever as the loop keeps running. Since we don't join, the thread will become detached
         // as the handle is dropped, and will be killed when the process/main thread exits.
     }
+
+    #[test]
+    fn test_block_unblock_signal() {
+        // Check if `SIGRTMIN() + 0` is blocked.
+        unsafe {
+            let mut sigset: sigset_t = mem::zeroed();
+            pthread_sigmask(SIG_BLOCK, null(), &mut sigset as *mut sigset_t);
+            assert_eq!(sigismember(&sigset, SIGRTMIN() + 0), 0);
```

Comment:
nit: It's cleaner to have a `let signal = SIGRTMIN() + 0;` somewhere and use that name throughout the code instead of using the addition each time.

Reply:
Sure, I will keep c_int and fix the cleaner nit.